zerver/webhooks/jira/view.py

```diff
@@ -44,6 +45,24 @@ def guess_zulip_user_from_jira(jira_username: str, realm: Realm) -> UserProfile
         return None
 
 
+def fetch_display_name_from_jira(account_id: str) -> str | None:
+    # Fetch the display name for a Jira account ID using Jira's API.
+
+    JIRA_API_URL = "https://your-jira-instance.atlassian.net/rest/api/2/user"
+    JIRA_AUTH = ("your-email@example.com", "your-api-token")  # Replace with actual credentials
```

Comment:
This is going to need to use the strategy from #29390 for managing the auth credentials. The code for this function needs to be the same for every user.

Reply:
Hi @timabbott 
Few things to note before proceeding, I see from [#29390] (https://github.com/zulip/zulip/pull/29390) that I need to make a `zulip_jira.py` script which takes these creds from command line.
But then we need to store those creds in a `config.json` file so if in future someone use mentions then API fetch occurs smoothly without re-entering the creds.

Also, I see in case of Trello, `zulip_trello.py` script is stored [python-zulip-api](https://github.com/zulip/python-zulip-api)](https://github.com/zulip/python-zulip-api/tree/main/zulip/integrations/trello) not in `zerver/webhooks/trello`.

Pls, guide should I proceed with making a zulip_jira.py script and storing creds in .json file in same folder `zerver/webhooks/jira`?

Looking forward to your feedback.

Reply:
#30465 might be the better example here instead of https://github.com/zulip/zulip/pull/29390, as it doesn't look like we need to create another standalone python script for this one.

Reply:
A point to note is that both use the same strategy to manage auth credentials by appending them as query parameters to the endpoint URL.